### PR TITLE
In-hand update, temperature gun update

### PR DESCRIPTION
**Pull request by Markolie**
Changes:
1) Attempts to clean up our in-hand system by updating it to /tg/'s: no more custom files for certain items. Instead there's now item_lefthand, item_righthand, clothing_lefthand, clothing_righthand, guns_lefthand and guns_righthand. 

Needs to be merged together with https://github.com/ParadiseSS13/Paradise/pull/2040 due to inhand naming changes.

*DISCLAIMER* 
Seeing as there were a million in-hands icon and I had to shift through each one manually, some in-hands may be MISSING. Please report them if you see them. If anyone feels like it, they can give the files I mentioned above a review to see if anything's in the wrong place.

2) In-hands can now by any size, by RemieRichards: https://github.com/tgstation/-tg-station/pull/11724.
3) Overhauls the temperature gun (by DeityLink): https://github.com/d3athrow/vgstation13/pull/1681. Here's a handy image explaining it:

### CODE REVIEW

File: code/game/objects/items/weapons/twohanded.dm [hunk 1]
```diff
@@ -546,9 +541,7 @@ obj/item/weapon/twohanded/
 /obj/item/weapon/twohanded/knighthammer
 	name = "singuloth knight's hammer"
 	desc = "A hammer made of sturdy metal with a golden skull adorned with wings on either side of the head. <br>This weapon causes devastating damage to those it hits due to a power field sustained by a mini-singularity inside of the hammer."
-
-	icon_override = 'icons/mob/in-hand/swords.dmi'
-	icon_state = "adrhammer0"
+	icon_state = "knighthammer0"
```
Comment (tigercat2000) [inline code/game/objects/items/weapons/twohanded.dm]: Why change this?

Comment (Markolie) [inline code/game/objects/items/weapons/twohanded.dm]: It's the only weapon in the game with a reference to one of Paradise's players, which is not something we want to do or encourage.

**Comment by TheDZD** [comment]
You are some kind of wizard. I cannot give this enough :+1:'s.

**Comment by KasparoVy** [comment]
Wow, that's absolutely wicked.

File: code/game/objects/items/weapons/twohanded.dm (hunk 1 above)
Comment (adrkiller59) [inline code/game/objects/items/weapons/twohanded.dm]: ..... :c 

Whatever, Im not gonna defend it. It was fun while it lasted. And no, i never asked or implemented that, if you were wondering.

File: code/modules/research/designs/weapon_designs.dm [hunk 2]
```diff
@@ -165,13 +165,12 @@
...
 /datum/design/temp_gun
 	name = "Temperature Gun"
-	desc = "A gun that shoots temperature bullet energy things to change temperature."//Change it if you want
+	desc = "A gun that changes the body temperature of its targets."
 	id = "temp_gun"
 	req_tech = list("combat" = 3, "materials" = 4, "powerstorage" = 3, "magnets" = 2)
 	build_type = PROTOLATHE
 	materials = list(MAT_METAL = 5000, MAT_GLASS = 500, MAT_SILVER = 3000)
 	build_path = /obj/item/weapon/gun/energy/temperature
-	locked = 1
```
Comment (Fox-McCloud) [inline code/modules/research/designs/weapon_designs.dm]: Err--it's unlocked now? Why so?

Comment (Markolie) [inline code/modules/research/designs/weapon_designs.dm]: It's unable to harm humans unless it's emagged (aside from 10 burn damage). This makes it a viable weapon for R&D to give to Xenobiology to fight slimes.

Comment (Fox-McCloud) [inline code/modules/research/designs/weapon_designs.dm]: Fair enough; thanks for the explanation.

**Comment by Fox-McCloud** [comment]
I literally cannot :+1:  x :100:  x :100:  This enough.

Thank you so so much for finally refactoring in-hands. It saves us a LOT of extra tedious work and resolves a very very very longstanding problem with our codebase. Thank you so very very much, @Markolie !

**Comment by Earthdivine** [comment]
Does Xenobio start with one or do they have to bug RnD to get the Temp gun?

**Comment by Markolie** [comment]
No, they do have to bug RnD. I don't mind giving them a freeze gun, although I'm not sure if that'll upset balance. Research is already quite powerful.

**Comment by Spacemanspark** [comment]
:+1:

**Comment by taukausanake** [comment]
@Markolie I agree to bug R&D for one. Fire extinguishers work fine with two hits to kill a slime.
That being said, we don't have those apparent extinguisher issues that /tg/ has/had? Also I'd like to ask about those slime grenades :3

**Comment by Markolie** [comment]
Yea, the image is just a copy paste from /vg/ - we may not have the same fire extinguisher issues.

As for slime grenades - you'll have to post a suggestion on the forums.

**Comment by SpaceTimeNow** [comment]
The weapon damage is based on temperature and can it can shot 3 x 12 heat damage on humans before it starts recharging for about 12 damage all 16 seconds.

This might be an issue with some of the races that have a lower "maxium heat values". I would assume they take more damge because they are further above their maximum. Also it mentions that people in Hardsuits take longer to normalize temerature and thereby will take more damage. So i am not quite sold on it being considered not dangerous. It should be tested against all races when it's ingame.

**Comment by Markolie** [comment]
The damage is dealt over several seconds. You'd have to shoot somone, wait for several seconds for the damage to apply and then shoot again. Even with double heat damage it'd be useless.

**Comment by SpaceTimeNow** [comment]
Temperature damage sounds like a bitch to balance and probably by now it works differently in many codebases, i am very interessted to see how this works out. 

The applications for that gun seem very interessting, probalby could use cryox without a cryo tube.

**Comment by Fox-McCloud** [comment]
@Markolie 

Which one of these should be merged first? This one of the guns one?

**Comment by Markolie** [comment]
The guns one.

**Comment by Fox-McCloud** [comment]
Merge conflicts =(

**Comment by Markolie** [comment]
Fixed.

**Comment by Fox-McCloud** [comment]
@Markolie 

Ready now (once it passes Travis)?

**Comment by Markolie** [comment]
Si

**Comment by Fox-McCloud** [comment]
Ruh-roh. Mapped items.

**Comment by Markolie** [comment]
Should be good if it passes now.

**Pull request merged**